### ISSUE TITLE
Inconsistency of return type

### ISSUE THREAD

**Issue by hanshino** [Kikobeats/human-number#5]
First of all, thanks a lot for developing this library.
My english is not good, so I used `DeepL` to help me write this issue.

As usage section, it will return `number` when `index === 0`. 

```js
const humanNumber = require("human-number");

let strNumber = humanNumber(100000);
let notStrNumber = humanNumber(10);

console.log(strNumber, "and type is", typeof strNumber);
console.log(notStrNumber, "and type is", typeof notStrNumber);
```

```bash
❯ node test.js
100K and type is string
10 and type is number
```

Would it be more consistent if all of them fixed the return string type?

**Comment by Kikobeats** [comment]
Fixed at v[1.0.8](https://github.com/Kikobeats/human-number/commit/2188d9880bc7072d3cdd4f1ccdef9303b7cfe63d) 🙂

**Comment by hanshino** [comment]
Thanks very much for such a quick fixed. Have a nice day😀
